aspnetcore/tutorials/grpc/grpc-client.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Tutorial: Create a .NET Core gRPC client"
+author: juntaoluo
+description: This series of tutorials shows how to create a gRPC Service on ASP.NET Core. Learn how to create a gRPC Service project, edit a proto file, and add an duplex streaming call.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: johluo
+ms.date: 4/10/2019
+uid: tutorials/grpc/grpc-client
+---
+# Tutorial: Create a .NET Core gRPC client
+
+By [John Luo](https://github.com/juntaoluo)
+
+This tutorial teaches how to create a .NET Core gRPC client that can communicate with gRPC services.
+
+At the end, you'll have a gRPC client that communicates with the gRPC Greeter service.
+
+[!INCLUDE[View or download sample code](~/includes/grpc/downloadClient.md)]
+
+In this tutorial, you:
+
+> [!div class="checklist"]
+> * Create a gRPC client.
+> * Run the service against the gRPC Greeter service created in the previous tutorial.
+> * Examine the project files.
+
+[!INCLUDE[](~/includes/net-core-prereqs-all-3.0.md)]
+
+## Create a .NET console application
+
+# [Visual Studio](#tab/visual-studio)
+
+Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio) to create a console app with the name GrpcGreeterClient.
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio-code) to create a console app with the name GrpcGreeterClient.
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/using-on-mac-vs-full-solution) to create a console app with the name GrpcGreeterClient.
+
+<!-- End of VS tabs -->
+
+---
+
+## Add required packages
+
+The following packages need to be added the gRPC client project:
+
+* [Grpc.Core](https://www.nuget.org/packages/Grpc.Core) which contains the C# API for the C-core client.
```

Comment:
Looking at other tutorials, I'd probably have something like [this](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/getting-started-with-swashbuckle.md#package-installation) for package installation.

Reply:
As seen in the user study, it doesn't hurt to have more detail here